### PR TITLE
feat: ENT-5689 | turn on subsidyRequest functionality when toggle bec…

### PR DESCRIPTION
**Pull request by christopappas**
…omes clickable

# For all changes

- [x] Ensure adequate tests are in place (or reviewed existing tests cover changes)

# Only if submitting a visual change

- [ ] Ensure to attach screenshots
- [ ] Ensure to have UX team confirm screenshots

### CODE REVIEW

File: src/components/settings/SettingsAccessTab/tests/SettingsAccessSubsidyRequestManagement.test.jsx [hunk 1]
```diff
@@ -50,6 +50,27 @@ describe('<SettingsAccessSubsidyRequestManagement />', () => {
     });
   });
 
+  it('should update configuration if disable = false and subsidy requests are not currently enabled ', async () => {
```
Comment (iloveagent57) [inline src/components/settings/SettingsAccessTab/tests/SettingsAccessSubsidyRequestManagement.test.jsx:53]: nit: `disabled`

**Review by iloveagent57 — APPROVED** [review]
Works on my machine.

File: src/components/settings/SettingsAccessTab/SettingsAccessSubsidyRequestManagement.jsx [hunk 2]
```diff
@@ -36,6 +36,12 @@ const SettingsAccessSubsidyRequestManagement = ({
     }
   }, [disabled, subsidyRequestsEnabled]);
 
+  useEffect(() => {
```
Comment (long74100) [inline src/components/settings/SettingsAccessTab/SettingsAccessSubsidyRequestManagement.jsx:39]: I think we want `if (!disabled && !subsidyRequestsEnabled)...` but keep the dependency array the same.

Comment (christopappas) [reply src/components/settings/SettingsAccessTab/SettingsAccessSubsidyRequestManagement.jsx:39]: for those watching at home, adding `!subsidyRequestsEnabled` will be prevent an API call if this is already enabled

**Review by long74100 — APPROVED** [review]

**Pull request merged**